aws_lambda_powertools/utilities/feature_flags/time_conditions.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime, timezone
+from typing import Dict, List
+
+from .schema import TimeValues
+
+HOUR_MIN_SEPARATOR = ":"
+
+
+def _get_utc_time_now() -> datetime:
+    return datetime.now(timezone.utc)
+
+
+def compare_between_utc_days(action: str, values: List[str]) -> bool:
+    current_day = _get_utc_time_now().strftime("%A").upper()
+    return current_day in values
+
+
+def compare_utc_datetime_range(action: str, values: Dict) -> bool:
+    current_time_utc: datetime = _get_utc_time_now()
+    start_date = datetime.strptime(values.get(TimeValues.START.value, ""), "%Y-%m-%dT%H:%M:%S%z")
+    end_date = datetime.strptime(values.get(TimeValues.END.value, ""), "%Y-%m-%dT%H:%M:%S%z")
+    return current_time_utc >= start_date and current_time_utc <= end_date
+
+
+def compare_utc_date_range(action: str, values: Dict) -> bool:
+    current_time_utc: datetime = _get_utc_time_now()
+    start_hour, start_min = values.get(TimeValues.START.value, HOUR_MIN_SEPARATOR).split(HOUR_MIN_SEPARATOR)
+    end_hour, end_min = values.get(TimeValues.END.value, HOUR_MIN_SEPARATOR).split(HOUR_MIN_SEPARATOR)
```

Comment:
I'm not sure I understand the logic here. If we don't have a START or END value, we're returning `":"`, which, when splitted, produces a tuple `("", "")`, which will crash on line 30.

Since the data is already validated, why don't we just return "" ?

Reply:
fixed